examples/features/torch_batch_process/batch_inference/compare_with_core_api/model.py

```diff
@@ -0,0 +1,53 @@
+import os
+from typing import Any, Dict, Optional, Sequence, Union
+
+import torch
+import torch.nn as nn
+
+from determined.experimental import client
+
+# Constants about the data set.
+IMAGE_SIZE = 32
+NUM_CHANNELS = 3
+NUM_CLASSES = 10
+
+TorchData = Union[Dict[str, torch.Tensor], Sequence[torch.Tensor], torch.Tensor]
+
+
+class Flatten(nn.Module):
+    def forward(self, *args: TorchData, **kwargs: Any) -> torch.Tensor:
+        assert len(args) == 1
+        x = args[0]
+        assert isinstance(x, torch.Tensor)
+        return x.contiguous().view(x.size(0), -1)
+
+
+model = nn.Sequential(
```

Comment:
I recommend not making this a singleton.  That's a bit odd to instantiate a deep learning model as a side-effect of importing a python module.

Have it returned from a function called `build_model()` to make it clear that calling the function results in creating a new model.

Then drop `get_model()` with the `determined_checkpoint = None` that isn't used by any of the code here.  It just distracts from the example.